### PR TITLE
net-p2p/airdcpp-webclient: bump to 2.2.0, fixes build with boost 1.65.0 + bump webui

### PR DESCRIPTION
**Pull request by sbraz**
Fixes https://bugs.gentoo.org/show_bug.cgi?id=630092

### CODE REVIEW

File: net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild [hunk 1]
```diff
@@ -20,13 +20,13 @@ RDEPEND="
 	dev-cpp/websocketpp
 	dev-libs/boost:=
 	dev-libs/geoip
-	dev-libs/leveldb
+	dev-libs/leveldb:=
```
Comment (a17r) [inline net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild]: I don't see subslots for that package, what was the reason for adding `:=`?

File: net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild [hunk 2]
```diff
@@ -20,13 +20,13 @@ RDEPEND="
...
 	dev-libs/openssl:0=[-bindist]
 	net-libs/miniupnpc:=
 	sys-libs/zlib
 	virtual/libiconv
-	nat-pmp? ( net-libs/libnatpmp )
-	tbb? ( dev-cpp/tbb )
+	nat-pmp? ( net-libs/libnatpmp:= )
```
Comment (a17r) [inline net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild]: I don't see subslots for that package, what was the reason for adding `:=`?

File: net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild [hunk 3]
```diff
@@ -20,13 +20,13 @@ RDEPEND="
...
+	tbb? ( dev-cpp/tbb:= )
```
Comment (a17r) [inline net-p2p/airdcpp-webclient/airdcpp-webclient-2.2.0.ebuild]: I don't see subslots for that package, what was the reason for adding `:=`?

**Comment by sbraz** [comment]
@a17r I added those following @mgorny's advice
```
21/06/2017 22:42 < sbraz> mgorny: do i need := when libs are slotted or all the time?
21/06/2017 22:42 <@mgorny> when they are slotted, subslotted or can become so in the reasonable future
21/06/2017 22:43 <@mgorny> (and you need to bind to specific slot/subslot)
21/06/2017 22:43 < sbraz> ok
21/06/2017 22:43 <@mgorny> for the 'can become' use common sense
21/06/2017 22:43 <@mgorny> if a package installs a single C/C++ library, then you can reasonably assume subslots will be used for SONAME at some point
```

**Comment by a17r** [comment]
But do you have reason to expect them to become slotted or subslotted in the foreseeable future? If only on the basis of installing a C/C++ library is too vague for my taste.

**Pull request closed without merging**